frontend/app/routes/$lang+/_public+/apply+/$id+/applicant-information.tsx

```diff
@@ -149,6 +149,7 @@ export default function ApplyFlowApplicationInformation() {
         <fetcher.Form method="post" aria-describedby="form-instructions-sin form-instructions-info" noValidate>
           <input type="hidden" name="_csrf" value={csrfToken} />
           <div className="mb-8 space-y-6">
+            <em id="name-instructions">{t('applicant-information.name-instructions')}</em>
```

Comment:
I think it should be wrap with a paragraph.

```suggestion
            <p id="name-instructions"><em>{t('applicant-information.name-instructions')}</em></p>
```